packages/react/src/components/ImageToolbar/ImagePanelController.tsx

```diff
@@ -11,14 +11,14 @@ import { FC } from "react";
 import { useBlockNoteEditor } from "../../hooks/useBlockNoteEditor";
 import { useUIElementPositioning } from "../../hooks/useUIElementPositioning";
 import { useUIPluginState } from "../../hooks/useUIPluginState";
-import { ImageToolbarProps } from "./ImageToolbarProps";
-import { ImageToolbar } from "./mantine/ImageToolbar";
+import { ImagePanelProps } from "./ImagePanelProps";
+import { ImagePanel } from "./mantine/ImagePanel";
 
-export const ImageToolbarController = <
+export const ImagePanelController = <
```

Comment:
Shouldn't this still be ImageToolbarController? It controls the toolbar right, not the panel? Maybe discuss on call